internal/ingress/controller/parser/parser.go

```diff
@@ -1499,10 +1499,17 @@ func (p *Parser) fillPlugins(state KongState) []Plugin {
 }
 
 func (p *Parser) globalPlugins() ([]Plugin, error) {
+	// removed as of 0.10.0
+	// only retrieved now to warn users
 	globalPlugins, err := p.store.ListGlobalKongPlugins()
 	if err != nil {
 		return nil, errors.Wrap(err, "error listing global KongPlugins:")
 	}
+	if len(globalPlugins) > 0 {
+		glog.Warning("global KongPlugins found. These are no longer applied as of and",
```

Comment:
> longer applied as of and

something is wrong

Reply:
It was indeed wrong. Added 0.10.0 as a stand-in version there. That is the upcoming `next` version, correct?

Reply:
Yes. In future, refrain from specifying versions though. Simply stating that a behavior has changed and asking users to refer to documentation is enough.